commands/bmc.nicole

```diff
@@ -191,11 +195,62 @@ function cmd_ifconfig {
 }
 
 # @doc cmd_syslog
-# Syslog server configuration
-#function cmd_syslog {
-#  not implemented
-#}
+# Remote logging settings
+function cmd_syslog {
+  subcommand "$@"
+}
 
+# @sudo cmd_syslog_show admin,operator,user
+# @doc cmd_syslog_show
+# Show actual remote syslog server
+function cmd_syslog_show {
+    local addr="$(busctl get-property ${SYSLOG_CONFIG_SERVICE} \
+                                      ${SYSLOG_CONFIG_PATH} \
+                                      ${NETWORK_CLIENT_IFACE} Address \
+                  2>/dev/null)"
+    addr=${addr#*\"}
+    addr=${addr%\"*}
+
+    local port="$(busctl get-property ${SYSLOG_CONFIG_SERVICE} \
+                                      ${SYSLOG_CONFIG_PATH} \
+                                      ${NETWORK_CLIENT_IFACE} Port \
+                  2>/dev/null)"
+    port=${port#* }
+
+    echo -n "Remote syslog server: "
+    if [[ -z "${addr}" ]] || [[ "${port}" == "0" ]]; then
+        echo "(none)"
+    else
+        echo -n "${addr}"
+        if [[ ! -z "${port}" ]]; then
+            echo -n ":${port}"
+        fi
+        echo
+    fi
+}
+
+# @sudo cmd_syslog_set admin
+# @doc cmd_syslog_set
+# Configure remote syslog server
+#  address[:port] - to change settings
+#  empty - to reset settings
```

Comment:
I would suggest to make a subcommand for resetting, lomething like `bmc syslog reset`.

Reply:
done